Bottom: e6346db38e72a8985e58da88719748a8bf460575
Top:    40f5d398b06b210099a271fdbc139c05545bcee4
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2016-03-09 12:41:51 -0300

Refresh of template-support.patch

---

diff --git a/libcc1/libcp1plugin.cc b/libcc1/libcp1plugin.cc
index 69c53f8..8bb413d 100644
--- a/libcc1/libcp1plugin.cc
+++ b/libcc1/libcp1plugin.cc
@@ -1352,7 +1352,7 @@ plugin_new_template_value_parm (cc1_plugin::connection *self,
   declarator.u.id.sfk = sfk_none;
 
   cp_decl_specifier_seq declspec;
-  clear_decl_specs (&declspec);
+  memset (&declspec, 0, sizeof (declspec));
   // cp_parser_set_decl_spec_type (&declspec, convert_in (type), -token-, false):
   declspec.any_specifiers_p = declspec.any_type_specifiers_p = true;
   declspec.type = convert_in (type);
